CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 2.0.6 (WIP)
```

Comment:
This will need to be 2.1.0 since it's a breaking compatibility change

Reply:
No problem.
Do you want me to tag it already as 2.1.0 or add a new one for future releases like `develop`?

Reply:
Sure you can change the version number. I'll remember to bump to dev tag when merging